### DISCUSSION TITLE
Resize Wezterm panes using Arrow Keys

### DISCUSSION THREAD

**Discussion by hadynz** [mrjones2014/smart-splits.nvim#226] [Q&A] [answered]
I would like to configure Wezterm to match how I have configured moving and resizing in Nvim, which is as follows:

- Moving: `CTRL` + `hjkl`
- Resizing: `CTRL` + `<Left> | <Down> | <Up> | <Right>`

Looking at the wezterm plugin options, and looking at the code, it doesn't seem that the above is possible. The configuration seems to make an assumption that `modifiers.move` and `modifiers.resize` must be different since they share the same `direction_keys`.

```
smart_splits.apply_to_config(config, {
  direction_keys = { 'h', 'j', 'k', 'l' },
  modifiers = {
    move = 'CTRL', -- modifier to use for pane movement, e.g. CTRL+h to move left
    resize = 'META', -- modifier to use for pane resize, e.g. META+h to resize to the left
  }
})
```

I almost want the opposite. I want both move/resize modifiers to be the same (`CTRL`), but use different keys for `direction_keys`.

Curious to know if I have misread code/documentation, and if someone has managed to manually.

**Comment by mrjones2014** [comment, marked as answer]
Hm, that's not a case I'd thought about.

I think to support your case in a backwards compatible way, I could update it to work like so:


```lua
smart_splits.apply_to_config(config, {
  -- if specified like this, share the same direction keys
  direction_keys = { 'h', 'j', 'k', 'l' },
  -- but you could also do this
  direction_keys = {
    move = { 'h', 'j', 'k', 'l' },
    resize = { 'Up', 'Down', 'Left', 'Right' },
  },
  modifiers = {
    move = 'CTRL', -- modifier to use for pane movement, e.g. CTRL+h to move left
    resize = 'META', -- modifier to use for pane resize, e.g. META+h to resize to the left
  }
})
```

**Reply by mrjones2014** [reply]
Added in #227

**Reply by hadynz** [reply]
Thanks. I'll try it out.

Do you know what's the best way to force download the latest using wezterm's plugin module?

**Reply by mrjones2014** [reply]
In wezterm’s command palette there’s an item to update all plugins
